src/test/jtx/mpt.h

```diff
@@ -97,7 +97,7 @@ struct MPTConstr
 
 struct MPTCreate
 {
-    std::optional<std::int64_t> maxAmt = std::nullopt;
+    std::optional<std::string> maxAmt = std::nullopt;
```

Comment:
Do we need to change the type? Can just change when the value is formatted to the field.